commitlint.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
       ['activity', 'bench', 'client', 'docs', 'core', 'release', 'samples', 'worker', 'workflow'],
     ],
     'header-max-length': [2, 'always', 120],
+    'body-max-line-length': [1, 'always', 120],
```

Comment:
Any chance you'd be willing to make this 100? Lines up with some other defaults and makes doing two-up views work nicely. Not a big deal.

Reply:
Made it a warning (`1`) because I needed to fit an extra long URL in the body: https://github.com/temporalio/sdk-node/pull/118/commits/6f8f168205443e9e5214e68820864fe9c9b615ca